src/io/pathHelper.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const electron = require('electron');
```

Comment:
This import should be in the `getAppdataPath()` function
The tests fail because they are not allowed to import `electron`